serialization.md

```diff
@@ -49,6 +50,10 @@ Address/Hash Helpers
       => #sender(#unparseByteStack(#parseHexBytes(#hashUnsignedTx(TN, TP, TG, TT, TV, #parseByteStackRaw(DATA), CID))), 28 -Int (TW %Int 2), #unparseByteStack(TR), #unparseByteStack(TS))
       requires TW ==Int CID *Int 2 +Int 35 orBool TW ==Int CID *Int 2 +Int 36
 
+    rule #sender(TYPE, CID, TN, TP, TG, TT, TV, TD, TA, TW, TR, TS)
```

Comment:
Is the `CID` argument here the same as the one for the other `#sender`s? Then lets put it in the same position, at the end of the argument list.